csharp/src/Google.Protobuf.Test/UnknownFieldSetTest.cs

```diff
@@ -138,6 +138,30 @@ public void TestClone(IMessage message)
             Assert.AreEqual(message.ToByteArray(), otherEmptyMessage.ToByteArray());
         }
 
+        [Test]
+        public void TestClone_lengthDelimited()
+        {
+            UnknownFieldSet destUnknownFieldSet = new UnknownFieldSet();
```

Comment:
nit: use `var destUnknownFieldSet = UnknownFieldSet()` instead of repeating the type name twice.  Also below.

Reply:
done